initfiles/examples/embed/couchbase-simple.ecl

```diff
@@ -0,0 +1,112 @@
+    },
+    "icao": "LFAC",
+    "id": 1254,
+    "type": "airport",
+    "tz": "Europe/Paris"
+*/
+/*
+  --sample airline record--
+        "callsign": null,
+        "country": "United States",
+        "iata": "WQ",
+        "icao": "PQW",
+        "id": 13633,
+        "name": "PanAm World Airways",
+        "type": "airline"
+*/
+
+dataset(travelrec) fulltravelrecords() := EMBED(couchbase : server(server), bucket(thebucket), detailed_errcodes(1), operation_timeout(5.5), config_total_timeout(15))
+    select mybucketalias.* from `travel-sample` as mybucketalias limit 500;
+ENDEMBED;
+
+dataset(namerec) airlinenames() := EMBED(couchbase : server(server), bucket(thebucket), detailed_errcodes(1), operation_timeout(5.5), config_total_timeout(15))
+    select mybucketalias.name from `travel-sample` as mybucketalias where mybucketalias.type = 'airline' limit 500;
+ENDEMBED;
+
+dataset(airportrec) airportrecords() := EMBED(couchbase : server(server), bucket(thebucket), detailed_errcodes(1), operation_timeout(5.5), config_total_timeout(15))
+    select mybucketalias.* from `travel-sample` as mybucketalias where mybucketalias.type = 'airport' limit 500;
+ENDEMBED;
+
+dataset(airportrec) usairportrecords() := EMBED(couchbase : server(server), bucket(thebucket), detailed_errcodes(1), operation_timeout(5.5), config_total_timeout(15))
+    select mybucketalias.* from `travel-sample` as mybucketalias where mybucketalias.type = 'airport' and country = 'United States' limit 500;
+ENDEMBED;
+
+dataset(airportrec) airportrecordsbycountry(string m) := EMBED(couchbase : server(server), bucket(thebucket), detailed_errcodes(1), operation_timeout(5.5), config_total_timeout(15))
+    select mybucketalias.* from `travel-sample` as mybucketalias where mybucketalias.type = 'airport' and country = $m limit 500;
+ENDEMBED;
+
+dataset(airportrec) airportrecordsbyid(integer id) := EMBED(couchbase : server(server), bucket(thebucket), detailed_errcodes(1), operation_timeout(5.5), config_total_timeout(15))
+    select mybucketalias.* from `travel-sample` as mybucketalias where mybucketalias.type = 'airport' and id = $id limit 500;
+ENDEMBED;
+
+dataset(airportrec) airportrecordsbyaltitude(integer alt) := EMBED(couchbase : server(server), bucket(thebucket), detailed_errcodes(1), operation_timeout(5.5), config_total_timeout(15))
+    select mybucketalias.* from `travel-sample` as mybucketalias where mybucketalias.type = 'airport' and geo.alt >= $alt limit 500;
+ENDEMBED;
+
+dataset(airportrec) airportrecordsbycoordinates(row(georec) values) := EMBED(couchbase : server(server), bucket(thebucket), detailed_errcodes(1), operation_timeout(5.5), config_total_timeout(15))
+  select mybucketalias.* from `travel-sample` as mybucketalias where mybucketalias.type = 'airport' and geo.lat = $lat and geo.lon = $lon limit 500;
+ENDEMBED;
+
+sequential
+(
+    OUTPUT(airlinenames());
+    OUTPUT(usairportrecords());
+    //OUTPUT(airportrecordsbycountry('United States'));
```

Comment:
Change this to make it work:

OUTPUT(airportrecordsbycountry('"United States"'));